.github/CONTRIBUTING.md

```diff
@@ -21,6 +21,11 @@ You are more than welcome to contribute to the system. This guide documents how
 -   To create a super user for the admin interface you can run
     `docker-compose run web ./manage.py createsuperuser`
 
+-   The see the emails we send (Payment confirmation etc.) run
```

Comment:
Forslår sætningen ændret til: `To see the emails generated by the system (Payment confirmations etc.) run`

Kan det disables igen med en tilsvarende kommando?

Reply:
Så er teksten rettet. 

Der er ikke behov for at disable den. 
Det er ment til lokal udvikling, den kører aldrig på prod/staging. 

Hvis jeg lokalt kører den så kommer mappen med emails. 
Den kan jeg så enten bare slette eller køre kommandoen igen for at få de nye